test/sql/simple/test_orderby.cpp

```diff
@@ -153,6 +153,27 @@ TEST_CASE("Test ORDER BY exceptions", "[order]") {
 
 	// and union itself fails if amount of entries is wrong
 	REQUIRE_FAIL(con.Query("SELECT a % 2, b FROM test UNION SELECT a % 2 AS k FROM test ORDER BY -1"));
+
+	// Top N queries that use ORDER_BY_LIMIT Optimization.
+	// Uses the new ORDER_BY_LIMIT optimization
+	result = con.Query("EXPLAIN SELECT a FROM test ORDER BY a LIMIT 1;");
```

Comment:
I think it is not a good idea to rely on EXPLAIN output in tests, instead it would be better to test functionality. If you must test whether or not optimizers work as expected create an optimizer test that inspects the internal nodes instead. This is done in e.g. `test_cse_optimizer.cpp`

Reply:
Agree. I will explore this possibility.